awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -513,6 +631,109 @@ function JobTemplateForm({
                 </FormCheckboxLayout>
               </FormGroup>
             </FormFullWidthLayout>
+            {enableWebhooks && (
+              <FormColumnLayout>
```

Comment:
All this webhook stuff feels like it is distinct enough (and also this file is getting so long), that I think that it would be both a UX and code improvement if we:
- create a sub form for this webhook related stuff
- moved all these fields (and related handlers) to a WebhookForm component

Reply:
yeah, had the same thought.  I can do that.  However I would like to do it as part of a different pr and I could do it to this form and to the workflowJobTemplateForm at the same time, as they are basically the same.

How does that sound to you?